docker/compose/smallbank-local-mounted.yaml

```diff
@@ -69,7 +69,7 @@ services:
       - "8080:8080"
     depends_on:
      - validator
-    command: sawtooth-rest-api -v --connect tcp://validator:4004 --bind rest-api:8080
+    command: sawtooth-rest-api -v --connect tcp://validator:4004 --bind rest-api:8080 --open_tsdb_url http://influxdb:8086 --open_tsdb_db metrics
```

Comment:
Command line arguments should never have underscores, and OpenTSDB seems to be one word.  Use dashes instead - "--opentsdb-url" and "--opentsdb-db".